src/pages/Home/components/FavoritesCitiesWeatherWidget/FavoritesCitiesWeatherWidget.tsx

```diff
@@ -1,16 +1,17 @@
 import { IconTypes } from 'enums/IconTypes';
+import { useLocalStorage } from 'hooks/useLocalStorage';
 import { CitiesWeatherWidget } from 'components/CitiesWeatherWidget';
 import { CityWeatherShortcut } from 'components/CityWeatherShortcut';
 import { FavoritesCitiesWeatherWidgetTestIds } from './FavoritesCitiesWeatherWidgetTestIds';
 
 const FavoritesCitiesWeatherWidget = () => {
-  const favoriteCities: string[] = [];
+  const [favoriteCities] = useLocalStorage<string[]>('favorites');
 
   return (
     <CitiesWeatherWidget data-testid={FavoritesCitiesWeatherWidgetTestIds.Container}>
       <CitiesWeatherWidget.Title>favorites</CitiesWeatherWidget.Title>
```

Comment:
**nitpick** I think that if there is no `favorites` then this title should not be displayed.

Reply:
Good point